server/interceptor/interceptor_test.go

```diff
@@ -278,21 +278,23 @@ func TestTrillianInterceptor_QuotaInterception(t *testing.T) {
 
 	ctx := context.Background()
 	for _, test := range tests {
-		qm := quota.NewMockManager(ctrl)
-		qm.EXPECT().GetUser(gomock.Any(), test.req).MaxTimes(1).Return(user)
-		if test.wantTokens > 0 {
-			qm.EXPECT().GetTokens(gomock.Any(), test.wantTokens, test.specs).Return(test.getTokensErr)
-		}
+		t.Run(test.desc, func(t *testing.T) {
+			qm := quota.NewMockManager(ctrl)
```

Comment:
I'd be better if `ctrl` was local to this test func, as then you'd get mock-related errors in the relevant test case, rather than at the end of all of the test cases (with no idea which caused the problem).

Reply:
Done.